models/QtiRunnerInitDataBuilder.php

```diff
@@ -181,7 +182,8 @@ protected function getTestMap(QtiRunnerServiceContext $context, array $itemsStat
 
                     $responsesCount = $this->getResponseCountsFromState($state);
 
-                    $isInformational = empty($state);
+                    $isUnseen = empty($state);
```

Comment:
I checked last change - it's good, except all informational items are now shown as 'unseen', even the ones I have already visited.
I can approve it, it's not a big deal for me, just check with QA.

![Screenshot_22](https://user-images.githubusercontent.com/38751932/149469433-56111f48-bbaf-4ad6-83bd-e35ecb12bab2.png)

Reply:
Also, if you compare with Solar, if you are e.g. on item-3 in normal launch and close browser tab without submitting anything, then in review mode this item will be shown as '_seen, but unanswered_' - which is logical.

Here such item will be shown as '_unseen_', which is not exactly not correct.

Again, I can approve it, not a big deal for me, just mentioning. 

![Screenshot_24](https://user-images.githubusercontent.com/38751932/149471079-99441a38-9cf4-4dbd-8e24-6b6c40bb2788.png)
![Screenshot_25](https://user-images.githubusercontent.com/38751932/149471084-b1de52df-974e-424d-afc9-979e74061b47.png)